samples/csharp/MicrosoftEduGraphSamples/Program.cs

```diff
@@ -14,5 +14,6 @@
 
 // Create an instance of the class you want to test and call the desired workflow method.
 // Each flow represents a common entire process that can be tested using Microsoft Graph SDK, all the sample flows are located in the "workflows" folder.
-ModuleWorkflow workflow = new ModuleWorkflow(config);
-await workflow.ClassworkAsync();
\ No newline at end of file
+
+SubmissionWorkflow workflow = new SubmissionWorkflow(config);
+await workflow.SubmissionFeedbackResource();
```

Comment:
SubmissionFeedbackResourceAsync (being consistent)

Reply:
Updated "SubmissionFeedbackResource" to "SubmissionFeedbackResourceAsync"